src/hooks/useKeyboardControls.ts

```diff
@@ -22,9 +23,11 @@ interface UseKeyboardControlsProps {
   isSeatTaken: (row: number, column: number) => boolean;
   takeSeat: (row: number | null, column: number | null) => void;
   enterSelectedRoom: () => void;
-  onMove?: () => void;
+  onMove?: () => void; // TODO: this doesnt seem to be needed anymore, can we remove it?
```

Comment:
- [ ] [TODO] this doesnt seem to be needed anymore, can we remove it?

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/840/commits/0fcc08e4ad8d690b9dcd1d9fdeef07ca79f88caa